WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -219,6 +219,9 @@
 
     <% end %>
 
+    <%= f.input :currency_code, collection: Money::Currency.table.keys %>
```

Comment:
I see on https://github.com/RubyMoney/money#currency that `currency.name` seems to be a thing. Could we present that to the user instead of the `iso_code`?